recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "pynvml" %}
-{% set version = "11.5.3" %}
-{% set python_min = "3.6" %}
+{% set version = "12.0.0" %}
+{% set python_min = "3.9" %}
```

Comment:
Since this is just [the default global Python min version]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/ce8474900318d755b579e233760d932941dcc587/recipe/conda_build_config.yaml#L826-L831 ), we can drop this

```suggestion
```